src/main/java/com/arpnetworking/tsdcore/sinks/HttpPostSink.java

```diff
@@ -122,6 +123,15 @@ protected Request createRequest(final AsyncHttpClient client, final byte[] seria
     protected URI getUri() {
         return _uri;
     }
+    
+    /**
+     * Accessor for the Apache <code>Uri</code>.
+     *
+     * @return The Apache <code>Uri</code>.
+     */
+    protected Uri getApacheUri() {
```

Comment:
nit: It's not for the Apache client.  It's for the asynchttpclient which is the next-iteration of the ning client.